data/scenarios/03challenges/test.yaml

```diff
@@ -1,20 +1,15 @@
-name: Moving, part 1
+name: Test challenge
 entities:
   - name: goal
     display:
       attr: device
       char: 'X'
     description:
-    - |
-      Robots can use the 'move' command to move forward one unit
-      in the direction they are currently facing.  To complete
-      this challenge, move your robot two spaces to the right, to
-      the coordinates (2,0).
+    - Pick a lambda.
```

Comment:
I tried to `pick` a lambda, but the command name is `grab`. :smile: Sounds better, shall we rename the command?

Reply:
Hah, well, but `pick` doesn't sound right for other things, e.g. `pick` a `tree`?  It's called `grab` because it's what you can do with a `grabber`.  In any case this challenge is just a placeholder for now.